spec/install/post_bundle_message_spec.rb

```diff
@@ -81,6 +81,22 @@
         expect(out).to include(bundle_complete_message)
       end
     end
+
+    describe "with misspelled or non-existent gem name" do
```

Comment:
I am :+1: on having one full test for this. It's been broken for many releases, I would like to make sure that doesn't happen again.